lib/archethic_web/templates/layout/root.html.eex

```diff
@@ -13,25 +13,29 @@
     <div class="container is-fluid">
       <nav class="navbar is-transparent" role="navigation" aria-label="main navigation">
         <div class="navbar-brand">
-            <a class="navbar-item is-size-5 is-uppercase" href="<%= Routes.live_path(@conn, ArchEthicWeb.ExplorerIndexLive) %>">
-              ArchEthic
-            </a>
-            <a role="button" class="navbar-burger burger" aria-label="menu" aria-expanded="false" data-target="navbar">
+          <a class="navbar-item is-size-5 is-uppercase" href="<%= Routes.live_path(@conn, ArchEthicWeb.ExplorerIndexLive) %>">
+            ArchEthic
+          </a>
+          <a role="button" class="navbar-burger burger" aria-label="menu" aria-expanded="false" data-target="navbar">
             <span aria-hidden="true"></span>
             <span aria-hidden="true"></span>
             <span aria-hidden="true"></span>
-            </a>
+          </a>
         </div>
-
         <div id="navbar" class="navbar-menu">
-            <div class="navbar-end">
-    <!-- 
-          TODO: implement the listing of transactions through BeaconChain
-
-              <a class="navbar-item" href="<%= Routes.live_path(@conn, ArchEthicWeb.TransactionListLive) %>">
-                 Transactions
+          <div class="navbar-end">
+            <%= if Mix.env() in [:dev, :test] or System.get_env("ARCHETHIC_NETWORK_TYPE") == "testnet" do %>
```

Comment:
Will this `Mix.env() in [:dev, :test]` work with release ?

Reply:
Yes since in release `System.get_env("ARCHETHIC_NETWORK_TYPE") == "testnet"`.

Reply:
I think you can use the layout view to create an helper to test the network type and use it to display or not the link

```elixir
defmodule ArchEthicWeb.LayoutView do
   def network_type do
      System.get_env("ARCHETHIC_NETWORK_TYPE") == "testnet"
   end
end
```

Reply:
Should I use ARCHETHIC_NETWORK_TYPE env variable or do I manage it from config?

Reply:
The best would be to use configuration to avoid direct call to System.